### ISSUE TITLE
Exporting E2E key for decrypting history in another client

### ISSUE THREAD

**Issue by Experimenter** [ajbura/cinny#178]
Please add a feature for exporting the E2E key from Cinny to decrypt the history in another client.
Eg: I am unable to read my sent messages when I logged into element.io
On re-requesting the keys nothing happens either.  

![Screenshot from 2021-12-05 21-34-19](https://user-images.githubusercontent.com/3885174/144754041-8deed529-daba-4de2-9a85-6c21b730421f.png)

**Comment by kfiven** [comment]
@Experimenter Just for more info, are you also using Secure backup to store your keys on server. If you are using it you can use your security key or paraphrase to restore those keys in element from server.

**Comment by Experimenter** [comment]
No, I was not using secure backup to store any keys. I was using nheko 
earlier and shifted to cinny. I imported the keys from nheko to cinny 
and logged out of nheko. Today I needed to create a private group with a 
primary address which could be accessed from other networks, for which I 
had to log into element.io. Thats when I found that the messages were 
encrypted. I thought I'd import the cinny keys to element but realised 
that it can't be done.
